tdp_vars_defaults/tdp-cluster/tdp-cluster.yml

```diff
@@ -35,8 +35,12 @@ exporter_livy_spark3_http_port: 18204
 #################################
 # Service & Component logs dirs #
 #################################
+tdp_python_date_pattern: "%Y-%m-%dT%H:%M:%S.%f%z"
```

Comment:
Will you need to add the following lines (this is done in the PR of Jupyterhub)?
```
tdp_date_iso8601_with_tz_enable: true
# If tdp_date_iso8601_with_tz_enable is false, %(asctime)s will be formatted with following pattern :
```

Reply:
Yes, this is done in the Jupyterhub PR. `tdp_date_iso8601_with_tz_enable` is not used by hue configuration. And the other line is a comment that does not make sense without the first line.